mempool/v0/clist_mempool_test.go

```diff
@@ -680,6 +681,38 @@ func TestMempoolRemoteAppConcurrency(t *testing.T) {
 	require.NoError(t, mp.FlushAppConn())
 }
 
+func TestRemoveBlobTx(t *testing.T) {
```

Comment:
Is this test supposed to be the same as `TestMalleatedTxRemoval`? in the v1 mempool?

Reply:
yeah, whoops, forgot to update this test. https://github.com/celestiaorg/celestia-core/pull/893/commits/f6f07fecab62abef28effbe02033518354db470f